tests/test_pipelines_common.py

```diff
@@ -29,7 +29,11 @@
 
 
 def get_checkpoint_from_architecture(architecture):
-    module = importlib.import_module(architecture.__module__)
+    try:
+        module = importlib.import_module(architecture.__module__)
+    except Exception:
```

Comment:
```suggestion
    except ImportError:
```
Let's try to have something not too broad here.